prime-angular-frontend/src/app/modules/phsa-labtech/pages/bcsc-demographic/bcsc-demographic.component.ts

```diff
@@ -36,7 +36,7 @@ export class BcscDemographicComponent implements OnInit {
     protected route: ActivatedRoute,
     protected router: Router,
     protected dialog: MatDialog,
-    protected phasLabtechResource: PhasLabtechResource,
+    protected phasLabtechResource: PhsaLabtechResource,
```

Comment:
Might want to change phasLabtechResource to phsaLabtechResource since IDE not smart enough to detect